docs/tutorials.rst

```diff
@@ -3,25 +3,20 @@
 Tutorials
 =========
 
+These tutorials cover basic usage of JAX and its features, including some of the
```

Comment:
The new tutorial ordering is more of a sequential flow. Could you update the text to this?

These tutorials cover basic usage of JAX and its features, including some of the
internal mechanisms that make JAX work. They start with the fundamentals and are meant to be read sequentially. For more in-depth discussions of JAX's
design and implementation, in no particular order, see the :doc:`advanced guides <advanced_guides>`.